src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Select, SelectOption,
+} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+
+/**
+ *
+ * @returns {object} Returns all of available cameras
+ */
+
+const getAvailableCameras = async (setDevices) => {
+  const cameraDevices = await navigator.mediaDevices.enumerateDevices()
+    .then((cameras) => cameras.filter((camera) => camera.kind === 'videoinput'));
+
+  setDevices([...cameraDevices]);
+};
+
+/**
+ * Webex Video Settings component
+ *
+ * @param {string} props.className  Custom CSS class to apply
+ * @returns {object} JSX of the component
+ */
+export default function WebexVideoSettings({className, meetingID}) {
+  const [devices, setDevices] = useState([]);
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-video-settings`;
+  const mainClasses = {
+    [classBaseName]: true,
+    [className]: !!className,
+  };
+
+  const extractCameraName = (label) => label.split('(', 1)[0];
```

Comment:
why is this needed? can you list an example output here please?

Reply:
Each camera has a code number at the end of its label and that's why I created this feature to remove it.

label: "Logitech HD Webcam C525 (046d:0826)" => Logitech HD Webcam C525